### PR TITLE
docs(README) Add to cite free-programming-books section

### PR DESCRIPTION
**Pull request by breck7**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,6 +215,24 @@ Volunteers have translated many of our Contributing, How-to, and Code of Conduct
 You might notice that there are [some missing translations here](docs/README.md#translations) - perhaps you would like to help out by [contributing a translation](docs/CONTRIBUTING.md#help-out-by-contributing-a-translation)?
 
 
-## License
-
-Each file included in this repository is licensed under the [CC BY License](LICENSE).
```
Comment (davorpa) [inline README.md:220]: I don't think that removing LIcense section will be a good option

File: README.md [hunk 2]
```diff
@@ -215,6 +215,24 @@ Volunteers have translated many of our Contributing, How-to, and Code of Conduct
...
+```
+
+```
+@article{pldb,
+  author = {Victor Felder al.},
+  title = {free-programming-books},
+  journal = {free-programming-books},
+  year = {2013},
+  note = {https://ebookfoundation.github.io/free-programming-books/}
+ }
```
Comment (davorpa) [inline README.md:237]: et al.

```suggestion
```
Victor Felder, et al. (2013) - "free-programming-books". Retrieved from: 'https://ebookfoundation.github.io/free-programming-books/' [Online Resource]
```

```
@article{pldb,
  author = {Victor Felder, et  al.},
  title = {free-programming-books},
  journal = {free-programming-books},
  year = {2013},
  note = {https://ebookfoundation.github.io/free-programming-books/}
 }
```

File: README.md [hunk 3]
```diff
@@ -215,6 +215,24 @@ Volunteers have translated many of our Contributing, How-to, and Code of Conduct
...
+https://ebookfoundation.github.io/free-programming-books/
+```
+
+```
+Victor Felder et al. (2013) - "free-programming-books". Retrieved from: 'https://ebookfoundation.github.io/free-programming-books/' [Online Resource]
+```
+
+```
+@article{pldb,
+  author = {Victor Felder al.},
```
Comment (davorpa) [inline README.md:232]:
```suggestion
  author = {Victor Felder, et al.},
```

File: README.md [hunk 4]
```diff
@@ -215,6 +215,24 @@ Volunteers have translated many of our Contributing, How-to, and Code of Conduct
...
+## To cite free-programming-books
+
+free-programming-books content is published to the public domain and you can use it freely. If needed, here are 3 options for citing free-programming-books:
+
+```
+https://ebookfoundation.github.io/free-programming-books/
+```
+
+```
+Victor Felder et al. (2013) - "free-programming-books". Retrieved from: 'https://ebookfoundation.github.io/free-programming-books/' [Online Resource]
```
Comment (davorpa) [inline README.md:227]:
```suggestion
Victor Felder, et al. (2013) - "free-programming-books". Retrieved from: 'https://ebookfoundation.github.io/free-programming-books/' [Online Resource]
```

**Comment by eshellman** [comment]
you can't just go and change a license. If you want to have a recommended citation format, please propose it in an issue, and explain why you think it's a good idea.

**Pull request closed without merging**